Source/Scene/PointCloudPostProcessor.js

```diff
@@ -95,65 +110,138 @@ define([
                 pixelDatatype : PixelDatatype.UNSIGNED_INT_24_8,
                 sampler : createSampler()
             });
+        }
 
-            // TODO : for now assume than any pass can write depth, possibly through EXT_frag_depth.
-            // Definitely needed for the initial render into the FBO, possibly also the ping-pong processing.
-            framebuffers[i] = new Framebuffer({
+        /* We want to reuse textures as much as possible, so here's the order
```

Comment:
Stick with the `//` comment style.

This is actually not in the Coding Guide, but I'm pretty sure it is a common practice throughout Cesium except for comments for generating reference doc with JSDoc.

Could you open a separate PR to mention this at the bottom of the [Basic Code Construction](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#basic-code-construction)?